sdk-extensions/zpages/README.md

```diff
@@ -1,34 +1,7 @@
 # OpenTelemetry SDK Extension zPages
 
-[![Javadocs][javadoc-image]][javadoc-url]
-
 This module contains code for the OpenTelemetry Java zPages, which are a collection of dynamic HTML
-web pages embedded in your app that display stats and trace data. Learn more in [this blog post][zPages blog];
-
-<!--- TODO: Update javadoc -->
-[javadoc-image]: https://www.javadoc.io/badge/io.opentelemetry/opentelemetry-sdk-contrib-auto-config.svg
-[javadoc-url]: https://www.javadoc.io/doc/io.opentelemetry/opentelemetry-sdk-contrib-auto-config
-[zPages blog]: https://medium.com/opentelemetry/zpages-in-opentelemetry-2b080a81eb47
-
-## Quickstart
```

Comment:
I don't think we need this for alpha modules, and the above was out of date too. Just went ahead and deleted to make the gradle simpler.